src/nginxconfig/generators/conf/proxy.conf.js

```diff
@@ -35,7 +35,12 @@ export default global => {
     config['proxy_set_header Connection'] = '$connection_upgrade';
     config['proxy_set_header Host'] = '$host';
     config['proxy_set_header X-Real-IP'] = '$remote_addr';
-    config['proxy_set_header X-Forwarded-For'] = '$proxy_add_x_forwarded_for';
+    config['proxy_set_header Forwarded'] = '$proxy_add_forwarded';
+    if (global.reverseProxy.proxyCoexistenceXForwarded.computed == 'passOn') {
+        config['proxy_set_header X-Forwarded-For'] = '$proxy_add_x_forwarded_for';
+    } else {
+        config['proxy_set_header X-Forwarded-For'] = '""';
+    }
     config['proxy_set_header X-Forwarded-Proto'] = '$scheme';
     config['proxy_set_header X-Forwarded-Host'] = '$host';
     config['proxy_set_header X-Forwarded-Port'] = '$server_port';
```

Comment:
Should these also be set to empty strings when the option is set to actively remove X-Forwarded-*, or should the option be reworded just be actively removing X-Forwarded-For?

Reply:
Based on [Forwarded format](https://datatracker.ietf.org/doc/html/rfc7239#section-4) should be empty; updated.